appinfo/info.xml

```diff
@@ -16,7 +16,7 @@
     <repository>https://github.com/nextcloud/files_rightclick</repository>
     <screenshot>https://raw.githubusercontent.com/nextcloud/files_rightclick/master/screenshots/picture.png</screenshot>
     <dependencies>
-        <nextcloud min-version="14" max-version="18" />
+        <nextcloud min-version="19" max-version="19" />
```

Comment:
this probably shouldn't be changed to min 19

Reply:
Yes, because this app is now shipped.
So we have one stable branch per release :)
https://github.com/nextcloud/files_rightclick/branches

Reply:
Yeah, sorry this has to be in sync to avoid future problems.